install.md

```diff
@@ -230,3 +230,113 @@ the [trouble shooting page](https://julialang.github.io/IJulia.jl/stable/manual/
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.6.1/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# Set the Julia variables that control the location of packages.
+# (Do not admit the current user's own depot path.)
+export JULIA_DEPOT_PATH=${central_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
+
+# Let Julia install and precompile the packages.
+${julia_for_oscar} -e 'using Pkg; \
+Pkg.add("Oscar"); using Oscar; \
+Pkg.add("GAP"); using GAP; \
+Pkg.add("Nemo"); using Nemo; \
+Pkg.add("Hecke"); using Hecke; \
+Pkg.add("Singular"); using Singular; \
+Pkg.add("Polymake"); using Polymake; \
+Pkg.add("AbstractAlgebra"); using AbstractAlgebra; \
+exit();'
+
+# Write a file that will be loaded by Julia on startup.
+echo 'using Pkg; Pkg.activate("'${central_depot}'/environments/v" * join(split(string(VERSION), ".")[1:2], "."))' > ${central_depot}/julia_load_initial.jl
+{% endhighlight %}
+</details>
+
+<details>
+<summary>
+Call Julia such that the system-wide installation of Oscar gets loaded
+when the user enters <code>using Oscar</code> in the Julia session.
+</summary>
+Enter the following commands into a file (for example <code>julia_with_oscar</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (not with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.6.1/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# Admit the user's own depot path and the path with the central installation.
+export JULIA_DEPOT_PATH=$HOME/.julia:${central_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
+
+# Call Julia.
+${julia_for_oscar} --load ${central_depot}/julia_load_initial.jl
+{% endhighlight %}
+</details>
+
+<details>
+<summary>
+Update the system-wide installation of Oscar when a new version is available.
+</summary>
+Enter the following commands into a file (for example <code>oscar_systemwide_update</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.6.1/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# Set the Julia variables that control the location of packages.
+export JULIA_DEPOT_PATH=${central_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
+
+# Let Julia update and precompile the installed packages.
+${julia_for_oscar} -e 'using Pkg; \
+Pkg.update(); \
```

Comment:
Won't this easily get stuck, as we saw last week, because all those dependencies are also added to the active environment?

Reply:
Perhaps it would be safer to just delete and recreate the environment? This way, more code could be shared with the first script, too

Reply:
Should we then say that updating means to remove the installation and to install it again, and thus no dedicated script for updating is needed at all?

Reply:
That would potentially break things for people who are running Julia/Oscar *right now*. This is just about the environment.

Reply:
O.k., if we remove the `Project.toml` before adding the packages then we also need just one script for installing and updating.

Reply:
@fingolfin I have updated the pull request in order to follow your idea.